### ISSUE TITLE
avoid repeating parts of specs that are already in the base NETCONF and QUIC

### ISSUE THREAD

**Issue by boucadair** [netconf-wg/netconf-over-quic#7]
I would also avoid repeating parts of specs that are already in the base NETCONF and QUIC. For example, there is no need to repeat all the same details about the Stream ID types, etc.

**Comment by marcblanchet** [comment]
I agree to make it as to the point as possible, and from your other comments, we did a lot. This one however seems to be relevant for the developer on what should be used for what, as discussed in the next paragraphs.

**Comment by marcblanchet** [comment]
closing for now.
